src/User.js

```diff
@@ -29,24 +31,65 @@ class User {
     this.recipesToCook.splice(index, 1); 
 	};
 
-	filterFavoriteRecipeByTag(tag) {
+	filterFavoriteRecipesByTag(tag) {
 		return this.favoriteRecipes.reduce((foundRecipes, recipe) => {
 			if (recipe.tags.includes(tag)) {
-				foundRecipes.push(recipe.id);
+				foundRecipes.push(recipe);
 			};
 			return foundRecipes;
 		}, []);
 	};
 
-	filterRecipeToCookByTag(tag) {
+	filterRecipesToCookByTag(tag) {
 		return this.recipesToCook.reduce((foundRecipes, recipe) => {
 			if (recipe.tags.includes(tag)) {
```

Comment:
Nicole - using includes here is AWESOME and allows for more flexibility when returning results. Great idea!!!